### PR TITLE
Lint tweaks

### PR DESCRIPTION
**Pull request by XhmikosR**
Requires #149. And there's still one lint error to fix in tests :)

I'll rebase this after the other PR is merged.

### CODE REVIEW

**Comment by bryanbraun** [comment]
#149 is merged, so this should be good to rebase. 👍

**Comment by XhmikosR** [comment]
Rebased. Only the tests error is left, feel free to push to my branch to fix it :)

BTW those (except for the `no-extend-native`) are not currently enforced. Let me know if you want me to have a look at enforcing them

File: .eslintrc.json
Comment (XhmikosR) [inline .eslintrc.json:10]: makes me wonder if the check is proper since `DocumentTouch` is not defined?

**Comment by bryanbraun** [comment]
@XhmikosR, I pushed up a fix to your branch and it looks like the tests are passing now. Feel free to pull this out of Draft whenever you think it's ready for review/merge.

**Comment by XhmikosR** [comment]
Where does `window.anchors` come from, though? Is it because you have it specified in globals maybe?

File: anchor.js
Comment (XhmikosR) [inline anchor.js:52]: https://github.com/Modernizr/Modernizr/blob/a0bab3c7bce629990acfaa67955632eeaee9f2e1/feature-detects/touchevents.js#L41

Comment (bryanbraun) [reply anchor.js:52]: Yeah, [this issue](https://github.com/bryanbraun/anchorjs/issues/47) has more background on why that feature was added.

That's interesting that Modernizr removed it. I'd be open to removing it at some point but it would be a breaking API change. I think I'd rather hold off unless I specifically hear reports of people having issues with it.

**Comment by bryanbraun** [comment]
> Where does window.anchors come from, though? Is it because you have it specified in globals at the top of the file maybe?

Yes, the relevant code is here: https://github.com/bryanbraun/anchorjs/blob/13e098f9c188ee485b13e46afcbf970a8c0a1b62/anchor.js#L19

There's no `window.anchors` if you're importing this via CommonJS (like you would with Webpack), but if you're loading the script tag in the browser, it creates a two global variables: one called `anchors` and one called `AnchorJS.`

File: .eslintrc.json
Comment (bryanbraun) [reply .eslintrc.json:10]: I believe it's needed because of [the code you pointed out below](https://github.com/bryanbraun/anchorjs/pull/154#pullrequestreview-496999725).

Comment (XhmikosR) [reply .eslintrc.json:10]: I know, but makes me wonder if the related code is correct since `DocumentTouch` isn't part of the `window` object. :)

File: anchor.js
Comment (XhmikosR) [reply anchor.js:52]: Perhaps just update it to the latest version before they removed it? And leave the removal for a major version.

File: anchor.js
Comment (bryanbraun) [inline anchor.js:241]: This is great... I didn't realize I could unescape these characters.

File: anchor.js
Comment (bryanbraun) [reply anchor.js]: Ok, I updated this to [the last version they used before removing the feature](https://github.com/Modernizr/Modernizr/pull/2432/files#diff-cf0b19caddca681af0bb076e85025f32L40), and pushed up the commit.

File: .eslintrc.json
Comment (bryanbraun) [reply .eslintrc.json:10]: Yeah it looks like [the feature was deprecated](https://developer.mozilla.org/en-US/docs/Web/API/DocumentTouch). I'm guessing it only remained in Modernizr's code for compatibility purposes.

**Comment by bryanbraun** [comment]
@XhmikosR, I think this just needs a rebase and to be pulled out of Draft status, and it'll be ready to merge.

**Comment by XhmikosR** [comment]
@bryanbraun Done. I made another deps PR and I think it should be ready after merging for a new release. Your call if it'll be a patch or minor bump :)

**Comment by XhmikosR** [comment]
Rebased it and remove the extra parens from the last patch, it should be good to go after a final review :)

**Review by bryanbraun — APPROVED** [review]
Looks great. Thanks!

**Comment by XhmikosR** [comment]
@bryanbraun can you release a new version too please? Minor or patch, it's your call :)

**Comment by bryanbraun** [comment]
@XhmikosR, ah yes, thanks for the reminder. I'll plan to do one tonight. 👍

**Comment by bryanbraun** [comment]
Just released 4.3.0 on github and npm: https://github.com/bryanbraun/anchorjs/releases/tag/4.3.0 🎉

**Pull request merged**
